pkgs/tools/security/tcb/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config
+, linux-pam, libxcrypt
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tcb";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "openwall";
+    repo = pname;
+    rev = "070cf4aa784de13c52788ac22ff611d7cbca0854";
+    sha256 = "sha256-Sp5u7iTEZZnAqKQXoPO8eWpSkZeBzQqZI82wRQmgU9A=";
+  };
+
+  outputs = [ "out" "bin" "dev" "man" ];
```

Comment:
What goes inside each of this directories? Usually `out` is used to be one of `bin` or `lib`.

Reply:
The `out` folder contains the main working libraries. The `bin` folder contains utilities that are used to convert the `/etc/shadow` file and are used once. There is no point in keeping them on the system all the time.